crates/test-case-macros/Cargo.toml

```diff
@@ -24,8 +23,8 @@ proc-macro = true
 path       = "src/lib.rs"
 
 [dependencies]
-cfg-if           = "1.0"
 proc-macro2      = { version = "1.0", features = [] }
 proc-macro-error = "1.0"
 quote            = "1.0"
 syn              = { version = "1.0", features = ["full", "extra-traits"] }
+test-case-utils = { version = "2.2.2", path = "../test-case-utils", default-features = false }
```

Comment:
It might be bit nitpicking:

I'm not sure if the `utils` prefix is unambiguous in that context.
Utils suggest that there are utilities, helpers, etc. While in this case, we put almost all test-case logic into it.
I would rather call the crate `test-case-core` . 

@luke-biel what do you think?

Reply:
your repo, all nit picks are welcome :) updated to `test-case-core`